client/rest-high-level/src/main/java/org/elasticsearch/client/MachineLearningClient.java

```diff
@@ -1877,4 +1882,129 @@ public void setUpgradeModeAsync(SetUpgradeModeRequest request, RequestOptions op
             listener,
             Collections.emptySet());
     }
+
+    /**
+     * Create a new data frame analytics config
+     * <p>
+     * For additional info
+     * see <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-put-calendar.html">
+     * ML create calendar documentation</a>
+     *
+     * @param request The request
+     * @param options Additional request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     * @return The {@link PutDataFrameAnalyticsResponse} containing the config
+     * @throws IOException when there is a serialization issue sending the request or receiving the response
+     */
+    public PutDataFrameAnalyticsResponse putDataFrameAnalytics(PutDataFrameAnalyticsRequest request,
+                                                               RequestOptions options) throws IOException {
+        return restHighLevelClient.performRequestAndParseEntity(request,
+            MLRequestConverters::putDataFrameAnalytics,
+            options,
+            PutDataFrameAnalyticsResponse::fromXContent,
+            Collections.emptySet());
+    }
+
+    /**
+     * Create a new machine learning calendar, notifies listener with the created calendar
```

Comment:
Calendar again. Same for get and delete below

Reply:
Done.